experimental/dds/tree2/src/test/objMerge.ts

```diff
@@ -45,13 +45,6 @@ export function merge<T>(lhs: T, rhs: T): Conflicted | Conflict | T {
 		return new Conflict(lhs, rhs);
 	}
 
-	// null is of type object, and needs to be treated as distinct from the empty object.
-	// Handling it early also avoids type errors trying to access its keys.
-	// Rationale: 'undefined' payloads are reserved for future use (see 'SetValue' interface).
-	if (lhs === null || rhs === null) {
-		return new Conflict(lhs, rhs);
-	}
-
```

Comment:
@yann-achard-MS I removed this so I could put a comment on it. What should we do about this?

Reply:
You can just remove the "Rationale" line. It doesn't make much sense in this context anymore.